internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
```

Comment:
Consider adding comments to the `Configuration` struct to describe what each field represents. This will improve code readability and maintainability.